src/renderers/shaders/ShaderLib/distanceRGBA_frag.glsl

```diff
@@ -9,6 +9,6 @@ void main () {
 
 	#include <clipping_planes_fragment>
 
-	gl_FragColor = packLinearUnitToRGBA( length( vWorldPosition.xyz - lightPos.xyz ) / 1000.0 );
+	gl_FragColor = packDepthToRGBA( length( vWorldPosition.xyz - lightPos.xyz ) / 1000.0 );
```

Comment:
I saw that. I think the shader is misusing the method. The argument should be in [0, 1], no? That argument only works on scenes of certain size units.